src/cli/cmd/iexec-result.js

```diff
@@ -239,8 +239,7 @@ pushSecret
           publicKeyName(address),
         );
       }
-      const publicKey = await readFile(secretFilePath, 'utf8');
-      const secretToPush = Buffer.from(publicKey, 'utf8').toString('base64');
```

Comment:
No need anymore to encode our PEM key into base64 here, that will now be done for us by `pushWeb2Secret()`.

Reply:
add key formatting again as the logic is no longer in the `pushWeb2Secret`